<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAB_MainWindow</class>
 <widget class="QMainWindow" name="DAB_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message de la banque :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOperation">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Opérations envoyées à la banque</string>
    </property>
    <widget class="QPushButton" name="pushButtonEnvoi">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoi</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>238</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonNumero">
          <property name="text">
           <string>Numéro de compte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMontant">
          <property name="text">
           <string>Montant :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditNumero">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMontant">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>66</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButtonSolde">
        <property name="text">
         <string>Solde</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRetrait">
        <property name="text">
         <string>Retrait</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDepot">
        <property name="text">
         <string>Dépot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEditEtat">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelAdresse">
         <property name="text">
          <string>Adresse :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPort">
         <property name="text">
          <string>Port : </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEditAdresse"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPort"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
